index.js

```diff
@@ -1,8 +1,18 @@
-import updateTerms from './src/index.js';
+import * as notifier from './src/notifier/index.js';
+import CGUs from './src/index.js';
 
 (async () => {
   try {
-    await updateTerms();
+    const app = new CGUs();
+    await app.init();
+
+    notifier.init(app.serviceProviders, app.documentsTypes);
+
+    app.on('sanitizedDocumentChange', notifier.onSanitizedDocumentChange);
+    app.on('documentScrapingError', notifier.onDocumentScrapingError);
+    app.on('applicationError', notifier.onApplicationError);
```

Comment:
Do we really want to do that in the `index.js` (as opposed to `scheduler`)? If I execute locally, I probably don't want to send notifications.

Reply:
Yep I agree.